src/stories/pagefold/Pagefold.tsx

```diff
@@ -1,22 +1,31 @@
 import React from "react";
 
-type PagefoldProps = {
+export type PagefoldProps = {
   inherit: boolean;
   container: boolean;
-  size: "small" | "medium" | "large" | "xlarge";
+  size: "none" | "xsmall" | "small" | "medium" | "large" | "xlarge";
+  colorClass?: "success" | "alert";
   children?: React.ReactNode;
   classes?: string;
```

Comment:
In React it is a convention to use the prop classNames for classes. I think you should not deviate from that.